.github/workflows/weekly.yml

```diff
@@ -82,7 +82,7 @@ jobs:
     name: "Q 2.7 M 21.3 image"
```

Comment:
While at it, could you remove this testing. There aren't expected any updates for 21.3/Q 2.7 so weekly testing seems overkill.